pkgs/applications/office/timeline/default.nix

```
@@ -0,0 +1,75 @@
+{ lib
+, fetchurl
+, python3Packages
+, makeDesktopItem
+, copyDesktopItems
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "timeline";
+  version = "2.4.0";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/thetimelineproj/${pname}-${version}.zip";
+    sha256 = "1pa0whqci6c2p20xf7gbndrrpr1xg42ixhql595ibdd4p3l37v23";
+  };
+
+  nativeBuildInputs = [ python3Packages.wrapPython copyDesktopItems ];
+
+  pythonPath = with python3Packages; [
+    wxPython_4_0
+    humblewx
+    icalendar
+    markdown
+    pysvg-py3
+    pillow
+  ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      desktopName = "Timeline";
+      name = "timeline";
+      comment = "Display and navigate information on a timeline";
+      icon = "timeline";
+      terminal = false;
+      exec = "timeline";
+      categories = "Office;Calendar;";
+    })
+  ];
+
+  dontBuild = true;
+  doCheck = false;
+
+  patchPhase = ''
+    sed -i "s|_ROOT =.*|_ROOT = \"$out/usr/share/timeline/\"|" source/timelinelib/config/paths.py
+  '';
+
+  installPhase = ''
+  runHook preInstall
+
+  site_packages=$out/${python3Packages.python.sitePackages}
+  install -D -m755 source/timeline.py $out/bin/timeline
+  mkdir -p $site_packages
+  cp -r source/timelinelib $site_packages/
+
+  mkdir -p $out/usr/share/timeline/locale
+  cp -r icons $out/usr/share/timeline/
+  cp -r translations/ $out/usr/share/timeline/
+
+  mkdir -p $out/share/icons/hicolor/{48x48,32x32,16x16}/apps
+  cp icons/48.png $out/share/icons/hicolor/48x48/apps/timeline.png
+  cp icons/32.png $out/share/icons/hicolor/32x32/apps/timeline.png
+  cp icons/16.png $out/share/icons/hicolor/16x16/apps/timeline.png
+
+  runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "http://thetimelineproj.sourceforge.net/";
+    changelog = "http://thetimelineproj.sourceforge.net/changelog.html";
+    description = "Display and navigate information on a timeline";
+    license = with licenses; [ gpl3Only cc-by-sa-30 ];
+    platforms = with platforms; linux ++ darwin;
```

```suggestion
    platforms = platforms.unix;
```
most likely works on all unix platforms.